packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -324,9 +331,18 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
       int totalNumber = 0;
       debugPrint('Some possible finders for the widgets at ${binding.globalToLocal(event.position)}:');
       for (Element element in candidates) {
-        if (totalNumber > 10)
+        if (totalNumber > 13)
```

Comment:
Maybe add a comment on how this number is derived?

Reply:
It's just whatever felt right to me...

Reply:
added a comment